### DISCUSSION TITLE
last.fm script help

### DISCUSSION THREAD

**Discussion by oakbloke** [jcorporation/mympd-scripts#27] [Q&A]
Hi all,
I have just installed myMPD and the last.fm script, created the variables and triggers outlined in the readme and tried to launch the script with trigger "key". I entered my username and password but I get an error "attempt to index a nil value" and it seems that the operation doesn't succeed. Is there something that I am missing? For reference this is running on an rpi5 under GentooPlayer. Thanks a lot!

**Comment by jcorporation** [comment]
You should find a more detailed error message in the log.

**Reply by oakbloke** [reply]
Thanks jcorporation, I tried to access the logs via ssh in gentooplayer but it said "journalctl: command not found". This is the notification I got when trying to run the script with trigger "key":

  | Error  executing script lastfm: Unknown error: [string "local function  hashRequest(data, secret)..."]:151: attempt to index a nil value (field  'var')

Apologies I'm not very good at coding things and spent another hour trying to get it to work and searching for answers.

**Reply by jcorporation** [reply]
Which myMPD version?

**Reply by oakbloke** [reply]
Version: 17.0.3

**Comment by jcorporation** [comment]
This version is too old for this script. Update always to latest myMPD version.

**Reply by oakbloke** [reply]
I updated via GentooPlayer and now the script works! Thanks a lot Jürgen :)
